bin/flume-ng

```diff
@@ -211,7 +211,11 @@ run_flume() {
   if [ ${CLEAN_FLAG} -ne 0 ]; then
     set -x
   fi
-  $EXEC $JAVA_HOME/bin/java $JAVA_OPTS $FLUME_JAVA_OPTS "${arr_java_props[@]}" -cp "$FLUME_CLASSPATH" \
+  
+  if [[ $FLUME_JAVA_OPTS =~ "Xmx" ]] || [[ $JAVA_OPTS =~ "Xmx" ]]; then
```

Comment:
We could make it more bulletproof by using `$FLUME_JAVA_OPTS =~ Xmx[0-9]+` check to make sure that there is at least one number after the `Xmx`.
Not sure that this is necessary though, what do you think, @chenjianwei3 ?